app/models/web/push_subscription.rb

```diff
@@ -84,9 +84,9 @@ def unsubscribe_for(application_id, resource_owner)
 
   def find_or_create_access_token
     Doorkeeper::AccessToken.find_or_create_for(
-      application: Doorkeeper::Application.find_by(superapp: true),
+      application: Doorkeeper::Application.webapp,
       resource_owner: user_id || session_activation.user_id,
-      scopes: Doorkeeper::OAuth::Scopes.from_string('read write follow push'),
+      scopes: Doorkeeper::Application.webapp.scopes,
```

Comment:
I'm not 100% sure why the scopes vary between the access token given to the browser and the one used for creating a push subscription? I would think these scopes should be the same?